src/components/shared/ReconfigurationDrawer.tsx

```diff
@@ -0,0 +1,147 @@
+import { t, Trans } from '@lingui/macro'
+import { Button, Drawer, DrawerProps, Input, Space } from 'antd'
+import { constants, utils } from 'ethers'
```

Comment:
We can potentially get big bundle size savings with this instead:

```suggestion
import * as constants '@ethersproject/constants'
import { BigNumber } from '@ethersproject/bignumber'
```